### DISCUSSION TITLE
AstroCalc Whats Up Tonight/WUT Tool API

### DISCUSSION THREAD

**Discussion by BlackBoxAstro** [Stellarium/stellarium#3515] [Q&A]
I am quite new to programming so still learning a lot. 

I am trying to create a python script to send the date, time and a location to Stellarium's WUT tool, and then export the output from specific lists (I am most interested in the Planets, Bright Nebulas, Galaxies, and Dark Nebula lists in the WUT tool).

I can't work out how/where to send the request (i.e. is there a way to access the tool from my localhost:8090/api/ ?)

I've tried looking all over and have read through the API docs but can't find what I am looking for. I have found this in the API docs but I don't see how to call the function (https://stellarium.org/doc/head/classAstroCalcDialog.html). Any help would be much appreciated!

**Comment by alex-w** [comment]
AstroCalc tools are not available via Remote Control API

**Reply by BlackBoxAstro** [reply]
Thank you Alex, I had thought that might be the case. Would you know a way I could accomplish what I am trying to achieve?

**Reply by gzotti** [reply]
Such function is not available until somebody extends the RC API. The RemoteControl API was created with dialog-less (w.r.t. the main window) interactive control of the program in mind, not acting as general-purpose web server for astronomical ephemeris computations.

**Reply by BlackBoxAstro** [reply]
Thank you Georg.
